scripts/configure-installer-tests-cluster.sh

```diff
@@ -13,8 +13,13 @@ KUBEADMIN_PASSWORD_FILE=${KUBEADMIN_PASSWORD_FILE:-"${DEFAULT_INSTALLER_ASSETS_D
 # Default values
 OC_STABLE_LOGIN="false"
 CI_OPERATOR_HUB_PROJECT="ci-operator-hub-project"
+# Copy kubeconfig to temporary kubeconfig file
+# Read, Write and Execute permission to temporary kubeconfig file
+mkdir -p ${DEFAULT_INSTALLER_ASSETS_DIR}/tmp/kubeconfig
+cp ${DEFAULT_INSTALLER_ASSETS_DIR}/auth/kubeconfig ${DEFAULT_INSTALLER_ASSETS_DIR}/tmp/kubeconfig
```

Comment:
wouldn't you delete the /auth/kubeconfig? wouldnt it be confusing to have two kubeconfigs?

Reply:
> wouldn't you delete the /auth/kubeconfig? wouldnt it be confusing to have two kubeconfigs?

This is original default kubeconfig provided by the installer. Due to write permission on the file in multistage test infra i need to copy that file to some location along with write permission. Then the new kubeconfig path will be updated through the env var KUBECONFIG.
Even i had the same thought to removing it, but as per my understanding though deleting the original kubeconfig brings no harm to the cluster setup but may be this original fresh kubeconfig can be used for some other purposes.